examples/capacitance/cap.lay

```diff
@@ -3,7 +3,7 @@
 $!VarSet |LFDSFN1| = '"cap2d.dat"'
 $!VarSet |LFDSVL1| = '"x" "y" "AtContactNode" "NSurfaceNormal_x" "NSurfaceNormal_y" "NodeVolume" "Potential" "Potential_prev" "SurfaceArea" "bot_bc" "bot_bc:Potential" "top_bc" "top_bc:Potential" "DField" "DField:Potential@n0" "DField:Potential@n1" "EdgeCouple" "EdgeInverseLength" "EdgeLength" "ElectricField" "ElectricField:Potential@n0" "ElectricField:Potential@n1" "Potential@n0" "Potential@n1" "unitx" "unity" "ElectricField_x" "ElectricField_y" "ElementEdgeCouple" "ElementNodeVolume"'
 $!SETSTYLEBASE FACTORY
-$!GLOBALLINKING 
+$!GLOBALLINKING
```

Comment:
I should probably remove this file, but I am just curious if the formatter actually understands at Tecplot layout file?